### PR TITLE
Useraccountpage dev

### PR DESCRIPTION
**Pull request by physgirl7**
Most recent update to the files

### CODE REVIEW

File: Pawfect-Harmony/build.gradle [hunk 1]
```diff
@@ -6,10 +6,8 @@ plugins {
 
 group = 'org.launchcode'
 version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
 
-java {
-	sourceCompatibility = '11'
```
Comment (EthanWillCode) [inline Pawfect-Harmony/build.gradle:11]: Why was this changed?
I want to know out of curiosity.

File: Pawfect-Harmony/src/main/java/org/launchcode/Pawfect/Harmony/PawfectHarmonyApplication.java [hunk 2]
```diff
@@ -4,11 +4,13 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 
-@SpringBootApplication (exclude = {DataSourceAutoConfiguration.class })
```
Comment (EthanWillCode) [inline Pawfect-Harmony/src/main/java/org/launchcode/Pawfect/Harmony/PawfectHarmonyApplication.java:7]: Same thing. Why was this changed? I am just curious.

**Review by EthanWillCode — APPROVED** [review]
Looks good to me!

I am approving this. 

There are a few changes that I am unsure of. Specifically the formatting change to java sourceCompatiblity and the change to @SpringBootApplication (exclude = {DataSourceAutoConfiguration.class }). Can you tell me more about these, please?

File: Pawfect-Harmony/build.gradle (hunk 1 above)
Comment (physgirl7) [reply Pawfect-Harmony/build.gradle:11]: For a while I was having trouble getting the program to build because of a connection to the MySQL database. I ended up trying a few things in the build.gradle file due to the error message being about "implementation 'mysql:mysql-connector-java'. The error said it couldn't get the library. At one point, the suggestion I was looking at had the java{sourceCompatibility = '11' where I presently put it in build. gradle. I'm not sure it really helped though. In the end the change build.gradle needed was "implementation 'mysql:mysql-connector-java:8.0.15'". We could probably switch the java source compatibility back down in the build.gradle to it's original position. I just haven't tried putting it back in it's earlier place to see if still works

File: Pawfect-Harmony/src/main/java/org/launchcode/Pawfect/Harmony/PawfectHarmonyApplication.java (hunk 2 above)
Comment (physgirl7) [reply Pawfect-Harmony/src/main/java/org/launchcode/Pawfect/Harmony/PawfectHarmonyApplication.java:7]: I had to take out the (exclude = {DataSourceAutoConfiguration.class }) in order to get the program to try to connect to a database. I had originally put that line in because I was trying to develop pages before I connected things to mySQL but I kept getting an error about not connecting to a database and I discovered that if you add that exclude statement after @SpringBootApplication it will keep it from trying to connect to a database. It was useful when I had no database, but I had to remove it once I got a database up and running to use.

Comment (EthanWillCode) [reply Pawfect-Harmony/src/main/java/org/launchcode/Pawfect/Harmony/PawfectHarmonyApplication.java:7]: You mentioned this in your slack messages, and talked about this several times in class. I should have reviewed them before looking at this pull request. I will do that in the future.

File: Pawfect-Harmony/build.gradle (hunk 1 above)
Comment (EthanWillCode) [reply Pawfect-Harmony/build.gradle:11]: Ok, I was just wondering.
I vote leave it alone.

Thank you for all of your hard work!
